configuration/pih/scripts/global/sampleCollection.js

```diff
@@ -0,0 +1,58 @@
+$(() => {
+	(async () => {
+	  // Sección: Arreglo de UI
+	  document.querySelector("#iframe > legend").innerText = "Solicitud de Análisis";
+
+	  // Sección: Llamadas a API - EMR
+	  const urlParams = new URLSearchParams(window.location.search);
+	  const encounterId = urlParams.get("encounterId"), patientId = urlParams.get("patientId");
```

Comment:
```suggestion
	  const encounterId = urlParams.get("encounterId");
	  const patientId = urlParams.get("patientId");
```

The comma operator should generally be avoided.